src/pages/StartFlowPage.tsx

```diff
@@ -47,7 +49,36 @@ export const StartFlowPage = () => {
         const loadedWorkflows = await WorkflowApi.getWorkflows()
         setWorkflows(loadedWorkflows)
         
-        if (loadedWorkflows.length > 0) {
+        if (loadedWorkflows.length === 0) {
+          // First session defaults
+          setDuration(Duration.fromObject({ minutes: 30 }))
+
+          try {
+            const defaultCategoryNames = ['social media', 'entertainment']
```

Comment:
I'd put the code for this try block in the BlockingPreferenceApi as `getDefaultSearchOptions`and do something like 
```
let defaultSearchOptions = BlockingPreferenceApi.getDefaultSearchOptions(params)
```
here. Should help with readability